src/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly make sure to add `rentals` into `attr_accessor` so it can have getters and setters
```suggestion
  attr_accessor :title, :author, :rentals
```